src/model/individual.coffee

```diff
@@ -1,4 +1,39 @@
+###
+ Object to work with $INDIVIDUAL
+ an object considered as $INDIVIDUAL is something like:
+
+ "payload":
+   {
+    "type": "$INDIVIDUAL",
+    "id": "ciptr4z1f00093k6krpx3vuhe",
+    "attributes":
+    {
+     "name": "Unnamed",
+     "type": "$INDIVIDUAL"
+    },
+   "relations": { }
+ }
+###
+
+
+
 module.exports =
   class Individual
 
-    constructor: (@id) ->
+    @id
+
+    constructor: (payload) ->
+      try
+        @id = payload.id
+      catch error
+        return null
```

Comment:
Its better to just let it fail so the user can follow the trace error instead of returning null